src/components/party/partyList/PartyList.tsx

```diff
@@ -1,179 +1,59 @@
-      },
-      {
-        memberId: 3,
-        nickname: '오예스3',
-        profileImage: 'https://bit.ly/ryan-florence',
-      },
-    ],
-    startDate: '2023-01-21',
-    endDate: '2023-03-22',
-    id: 3,
-  },
-  {
-    coverImage: 'https://via.placeholder.com/700x500',
-    name: '퇴사기념',
-    description:
-      '설명은 필요없다. 떠나자. 설명은 필요없다. 떠나자. 설명은 필요없다. 떠나자. 설명은 필요없다. 떠나자.',
-    members: [
-      {
-        memberId: 1,
-        nickname: '민재',
-        profileImage: 'https://bit.ly/broken-link',
-      },
-      {
-        memberId: 2,
-        nickname: '유리',
-        profileImage: 'https://bit.ly/broken-link',
-      },
-      {
-        memberId: 3,
-        nickname: '지영',
-        profileImage: 'https://bit.ly/broken-link',
-      },
-      {
-        memberId: 3,
-        nickname: '천욱',
-        profileImage: 'https://bit.ly/broken-link',
-      },
-    ],
-    startDate: '2023-02-21',
-    endDate: '2023-02-22',
-    id: 4,
-  },
-  {
-    coverImage: 'https://via.placeholder.com/700x500',
-    name: '취업기념',
-    description: '우모 취업기념 모임, 다들 취할 준비들 해',
-    members: [
-      {
-        memberId: 1,
-        nickname: '오예스1',
-        profileImage: 'https://bit.ly/ryan-florence',
-      },
-      {
-        memberId: 2,
-        nickname: '오예스2',
-        profileImage: 'https://bit.ly/sage-adebayo',
-      },
-      {
-        memberId: 3,
-        nickname: '오예스3',
-        profileImage: 'https://bit.ly/code-beast',
-      },
-      {
-        memberId: 4,
-        nickname: '오예스4',
-        profileImage: 'https://bit.ly/code-beast',
-      },
-      {
-        memberId: 5,
-        nickname: '오예스5',
-        profileImage: 'https://bit.ly/code-beast',
-      },
-      {
-        memberId: 6,
-        nickname: '오예스6',
-        profileImage: 'https://bit.ly/code-beast',
-      },
-      {
-        memberId: 7,
-        nickname: '오예스7',
-        profileImage: 'https://bit.ly/code-beast',
-      },
-    ],
-    startDate: '2023-04-21',
-    endDate: '2023-06-22',
-    id: 5,
-  },
-];
+import PartyListCard from './PartyListCard';
 
 const PartyList = () => {
-  const onGoing = DUMMYDATA.filter((party) => new Date(party.endDate) >= new Date());
-  const expired = DUMMYDATA.filter((party) => new Date(party.endDate) < new Date());
+  const {
+    data: onGoingPartyList,
+    isLoading: onGoingLoading,
+    isError: onGoingError,
+  } = useQuery<MyPartyList>(
+    ['onGoingPartyList'],
+    () => fetchMyPartyList({ partyType: 'ONGOING', pageSize: 1000 }),
+    {
+      staleTime: 10000,
+    }
+  );
+
+  const {
+    data: completedPartyList,
+    isLoading: completedLoading,
+    isError: completedError,
+  } = useQuery<MyPartyList>(
+    ['completedPartyList'],
+    () => fetchMyPartyList({ partyType: 'COMPLETED', pageSize: 1000 }),
+    {
+      staleTime: 10000,
+    }
+  );
+
+  if (onGoingLoading) return <></>;
+  if (completedLoading) return <></>;
+  if (onGoingError) return <></>;
+  if (completedError) return <></>;
```

Comment:
```suggestion
  if (onGoingLoading || completedLoading) return <></>;
  if (onGoingError || completedError) return <></>;
```
약소하지만... 이런 식으로 줄일 수 있을 것 같아요!